### ISSUE TITLE
horizontalEdge operator math shifts constraints

### ISSUE THREAD

**Issue by KingOfBrian** [Raizlabs/Anchorage#3]
horizontalEdge operator shifts the constraint, so:

```swift
view.horizontalAnchors == horizontalAnchors + 10
```

This shifts the leading and trailing constraint right by 10. This makes sense, but isn't overly useful in practice. It feels like EdgeAnchors or horizontal / vertical anchors should inset the anchors so +10 would shift the leading left and the trailing right.

**Comment by dostrander** [comment]
I was just about to propose this.

Or another solution is we add a `+-` operator to `edgeAnchors` which will basically be a padding operator.  
```swift
view.horizontalAnchors == horizontalAnchors +- 10 // pad the horizontal anchors with 10 pts
```

We could extend this to be more of a "smart" operator and which will shift the order of the views around to always makes the a positive value be padding between views.

For example currently
```swift
view1.bottomAnchor == view2.topAnchor + 10 // this actually just pushes the bottom past the top of the second view by 10
```

```swift
view1.bottomAnchor == view2.topAnchor +- 10 // this would pad the bottom anchor
```

**Comment by dostrander** [comment]
Sorry wrong button, was just going to propose this

**Comment by ahtierney** [comment]
It would be cool to change the way edge anchors work and use inset/padding or outset/margin style operators `<=>` or `>=<` to define insets or outsets respectively. If the right/left/up/down push syntax makes sense you could do a shift operator (idk like: `=>>`) or something that's less of a trick of assumptions.

**Comment by jvisenti** [comment]
Fixed in #4, incorporated in `2.0.0`
